Transfer/Sources/Helper/TransferAmountCurrencyFormatter.swift

```diff
@@ -21,7 +21,8 @@ import Foundation
 import Common
 #endif
 
-struct TransferAmountCurrencyFormatter {
+/// Transfer Amount Currency Formatter 
+public struct TransferAmountCurrencyFormatter {
```

Comment:
Move this to common module as well since now it's being used in other modules as well

Reply:
Updated